src/components/Offices/Identity/index.js

```diff
@@ -115,6 +116,33 @@ function IdentityAnalysis({ identity }) {
   );
 }
 
+function TokenTable({ llmBalance }) {
+  return (
+    <Table
+      columns={[
+        {
+          Header: "User balance in centralized database",
+          accessor: "desc",
+        },
+        {
+          Header: "",
+          accessor: "res"
+        }
+      ]}
+      data={[
+        {
+          "desc": "LLM balance",
+          "res": llmBalance ? Number(llmBalance)/ 10**12 : null,
+        },
+        {
+          "desc": "LLD balance",
+          "res": llmBalance ? Number(llmBalance)/ 10**11 : null,
```

Comment:
No. Use ethers.utils.formatUnits for formatting or something from wallet utils.

Reply:
Also: keep in mind that LLD has 12 decimal points, same as LLM. It's just that we're giving users 10x as many dollars as merits.